sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -49,10 +60,13 @@ def __init__(self, **kwargs):
         self.created_on = kwargs.get("created_on", None)
         self.digest = kwargs.get("digest", None)
         self.last_updated_on = kwargs.get("last_updated_on", None)
-        self.manifest_properties = kwargs.get("manifest_properties", None)
         self.operating_system = kwargs.get("operating_system", None)
+        self.references = kwargs.get("references", None)
         self.size = kwargs.get("size", None)
         self.tags = kwargs.get("tags", None)
+        self.content_permissions = kwargs.get("content_permissions", None)
```

Comment:
I believe `get` already defaults to `None` if key is not in there, so second arg `None` is not needed.
(not saying this needs to be changed or anything)